### PR TITLE
add logging api

### PR DESCRIPTION
**Pull request by yigal-dayan2**

### CODE REVIEW

File: Source/IBMCloudAppID/internal/TokenManager.swift [hunk 1]
```diff
@@ -298,4 +298,46 @@ internal class TokenManager {
         self.latestRefreshToken = nil
     }
 
+    internal func sendLoggingRequest(aToken:AccessToken?, iToken:IdentityToken?, activity:String) {
+        if (aToken == nil || iToken == nil) {
```
Comment (rotembr) [inline Source/IBMCloudAppID/internal/TokenManager.swift]: change the names to idToken and accessToken

File: Source/IBMCloudAppID/internal/TokenManager.swift [hunk 2]
```diff
@@ -298,4 +298,46 @@ internal class TokenManager {
...
+
+        let internalCallback: BMSCompletionHandler = {(response: Response?, error: Error?) in
+            if error != nil {
+                TokenManager.logger.debug(message: "Error sending logging request");
+            } else {
+                TokenManager.logger.debug(message: "OK sending logging request");
+            }
+        }
+
+        let request = Request(url: loggingUrl, method: HttpMethod.POST, headers: headers, queryParameters: nil, timeout: 0)
```
Comment (rotembr) [inline Source/IBMCloudAppID/internal/TokenManager.swift]: use appid request module

File: Source/IBMCloudAppID/internal/TokenManager.swift [hunk 3]
```diff
@@ -298,4 +298,46 @@ internal class TokenManager {
...
+        request.timeout = BMSClient.sharedInstance.requestTimeout
+
+        // body.data(using: .utf8)*/
+        sendRequest(request: request, body: body, internalCallBack: internalCallback)
+    }
+    public func notifyLogout() {
+
+        sendLoggingRequest(aToken:self.latestAccessToken, iToken:self.latestIdentityToken, activity:"logout");
```
Comment (rotembr) [inline Source/IBMCloudAppID/internal/TokenManager.swift]: change activity to event name

File: Source/IBMCloudAppID/internal/TokenManager.swift (hunk 2 above)
Comment (yigal-dayan2) [reply Source/IBMCloudAppID/internal/TokenManager.swift]: i used what the other calls did, i think they had some problem posting jsons with the other module

**Pull request merged**